pypeit/reduce.py

```diff
@@ -190,17 +198,117 @@ def parse_manual_dict(self, manual_dict, neg=False):
         # Return
         return manual_extract_dict
 
-    def find_objects(self, image, std=False, ir_redux=False, std_trace=None, maskslits=None,
-                     show_peaks=False, show_fits=False, show_trace=False, show=False,
-                     manual_extract_dict=None, debug=False):
+    def extract(self):
```

Comment:
Let's talk through these changes. It is difficult to review this restructuring on GitHub.

Reply:
Done.